include/seqan3/alphabet/aminoacid/aa10murphy.hpp

```diff
@@ -0,0 +1,240 @@
+ * A represents alanine, as well as the two additional, species-specific aminoa acids Selenocysteine and Pyrrolysine.
+ * B represents charged/polar residues (E,D,N,Q).
+ * F represents amino acids with large and mainly hydrophobic aromatic side chains (F,W,Y).
+ * I represents large hydrophobes (L,V,I,M).
+ * K represents long-chain positively charged residues (K,R).
+ * S represents alcohols (S,T).
+ * C, G, H and P do not represent any other amino acids other than themselves.
+ *
+ * The alphabet may be brace initialized from the static letter members (see above). Note that you cannot assign
+ * regular characters, but additional functions for this are available.
+ *
+ * This alphabet allows to reduce the aminoacid alphabet size to 10 but is still able to recognize and represent
+ * folding of all proteins. Amino acids are grouped together based on similar physical and chemical properties.
+ *
+ * *Note:* Letters which belong in the extended alphabet will be automatically converted.
+ * Terminator characters are converted to F, because the most commonly occurring stop codon in higher eukaryotes
+ * is UGA <sup>2</sup>. This is most similar to a Tryptophan which in this alphabet
+ * gets converted to Phenylalanine. Anything unknown is converted to A.
+ *
+ * |Input Letter  |Converts to    |
+ * |--------------|---------------|
+ * |D             |B<sup>1</sup>  |
+ * |E             |B<sup>1</sup>  |
+ * |J             |I<sup>1</sup>  |
+ * |L             |I<sup>1</sup>  |
+ * |M             |I<sup>1</sup>  |
+ * |N             |B<sup>1</sup>  |
+ * |O             |A<sup>1</sup>  |
+ * |Q             |B<sup>1</sup>  |
+ * |R             |K<sup>1</sup>  |
+ * |T             |S<sup>1</sup>  |
+ * |U             |A<sup>1</sup>  |
+ * |V             |I<sup>1</sup>  |
+ * |W             |F<sup>1</sup>  |
+ * |Y             |F<sup>1</sup>  |
+ * |Z             |B<sup>1</sup>  |
+ * |X (Unknown)   |A<sup>1</sup>  |
+ * |* (Terminator)|F<sup>1,2</sup>|
+ *
+ * <sup><b>1</b></sup>L. R. Murphy, A. Wallqvist, and R. M. Levy. Simplified amino acid alphabets for protein
+ * fold recognition and implications for folding. Protein Eng., 13(3):149–152, Mar 2000.\n
+ * <sup><b>2</b></sup>Trotta, E. (2016). Selective forces and mutational biases drive stop codon usage
+ * in the human genome: a comparison with sense codon usage.
+ * BMC Genomics, 17, 366. http://doi.org/10.1186/s12864-016-2692-4
+ *
+ * \snippet test/snippet/alphabet/aminoacid/aa10murphy.cpp construction
+ */
+class aa10murphy : public aminoacid_base<aa10murphy, 10>
+{
+private:
+    //!\brief The base class.
+    using base_t = aminoacid_base<aa10murphy, 10>;
+
+    //!\brief Befriend seqan3::aminoacid_base.
+    friend base_t;
+    //!\cond \brief Befriend seqan3::alphabet_base.
+    friend base_t::base_t;
+    //!\endcond
+
+public:
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    constexpr aa10murphy() : base_t{} {}                      //!< Defaulted.
```

Comment:
can you `=default` this as well?

And properly align the comments please :nail_care: